src/hooks/protections/receive_net_message.cpp

```diff
@@ -64,7 +65,7 @@ namespace big
 						buffer.ReadString(message, 256);
 						buffer.ReadQWord(&unk, 64);
 						buffer.ReadBool(&is_team);
-						LOG(INFO) << "[CHAT] from " << player->get_name() << ": " << message << (is_team ? " [TEAM]" : " [ALL]");
+						LOG(INFO) << "[CHAT] from " << player->get_name() << ": " << message << (is_team) ? " [TEAM]" : " [ALL]";
```

Comment:
This adds weird numbers at the ends of messages:
```
[17:58:38][INFO/receive_net_message.cpp:68]: [CHAT] from no2: АХАХХАХАХААХ247
[17:58:40][INFO/receive_net_message.cpp:68]: [CHAT] from no1: ну го на снайпах0
[17:58:43][INFO/receive_net_message.cpp:68]: [CHAT] from no2: бездарность 0
[17:58:47][INFO/receive_net_message.cpp:68]: [CHAT] from no2: хааххаха0
[17:58:52][INFO/receive_net_message.cpp:68]: [CHAT] from no2: удачи хуйлуша 0
```
Sorry for a bad example.

Also this `is_team` gives a lot of false positives and probably isn't actually `is_team`.

Reply:
This is what IDA shows for the game parsing the CMsgTextMessage, it seems to read a bool of some sorts.

![image](https://user-images.githubusercontent.com/24669514/202530893-f5e91332-05d6-400e-8111-161eb96b5d0a.png)